### PR TITLE
OAuth Redirect with url parameters over GET

### PR DESCRIPTION
**Pull request by nazarhussain**

### CODE REVIEW

File: oauth_manager.go [hunk 1]
```diff
@@ -146,10 +146,20 @@ func (o *OAuthHandlers) HandleAuthorizePassthrough(w http.ResponseWriter, r *htt
 			fmt.Fprintf(w, string(responseMessage))
 			return
 		}
-
-		w.Header().Add("Location", o.Manager.API.Oauth2Meta.AuthorizeLoginRedirect)
+		if r.Method == "GET" {
+			var buffer bytes.Buffer
```
Comment (lonelycode) [inline oauth_manager.go]: need to add an import for "bytes" at the top, build is failing :-)

Comment (nazarhussain) [inline oauth_manager.go]: Done. Build passed.

**Comment by lonelycode** [comment]
Smashing, thanks! :-)

**Comment by nazarhussain** [comment]
I would look forward to get this change in next release so I can update my installation to use it. When it is expected the next release?

**Comment by lonelycode** [comment]
Hmmm, no release slated until April AFAIK, you'd be best compiling your PR yourself :-)

**Pull request merged**